File: Practica1/Ej4/Cliente.java
class Cliente{

    private FilterManager filtermanager;

    public Cliente(FilterManager f){
            this.filtermanager=f;
    }

    public void crearCuenta(String correo, String contra){
        Cuenta cuenta=new Cuenta(correo,contra);
        filtermanager.haceCuenta(cuenta);
    }

}

File: Practica2/Ej1/lib/Cliente.dart
import 'FilterManager.dart';
import 'Cuenta.dart';
class Cliente {

  FilterManager? _filterManager;

  Cliente (FilterManager f){
    _filterManager = f;
  }

  String crearCuenta(String correo, String contra){
    Cuenta cuenta = Cuenta(correo, contra);
    return _filterManager!.haceCuenta(cuenta);
  }


}